### DISCUSSION TITLE
unresponsive at prompt?

### DISCUSSION THREAD

**Discussion by skytwosea** [llm-workflow-engine/llm-workflow-engine#83] [General]
Successfully installed last night. This afternoon, I have gone through the steps to activate the shell environment, but no response is delivered when submitting a prompt.
1. activate my conda env & run `chatgpt install`
2. sign in, click through prompts in browser, reach browser prompt. Browser prompt works.
3. close browser window, !exit shell environment, restart shell env without `install` parameter. Program opens and sets a prompt: `1>`
.. but the prompt is unresponsive; nothing is returned when the prompt is submitted at `enter`.

I've restarted the shell, the conda environment, and my machine, and have updated the repo via `git pull origin main`, all to no avail; same problem, just completely unresponsive.

Any tips or suggestions? Any log files I can provide?
Kubuntu 5.15.0

**Comment by Tecuya** [comment]
Recently ChatGPT changed a parameter in their request, and I made a new version to be compatible with that change.  Now they have changed that parameter back, so once again I've made a new version to restore functionality.  Please update and try again

**Reply by tpseers** [reply]
@Tecuya It seems not working (with the latest version), can you please confirm?

**Comment by skytwosea** [comment]
@Tecuya oh jeez, how frustrating! Growing pains. Will do, and thanks

**Comment by Tecuya** [comment]
OpenAPI has two things coming that might change things for this project and improve the reliability of chatgpt-wrapper

* They have been on record saying they want to make a version of their "conversations" API (basically the thing this project uses) part of their public API offering (currently it is not).  If they do this, we can make a version of this program that doesn't need playwright, but open AI charges you for the API  use.

* Depending on how ChatGPT plus works, it may allow this wrapper to have a much more streamlined experience, hopefully one that doesn't require you to re-login every time you want to actually use the thing, which is the main reason I don't use this project more in my daily work.

For either of these, this project is set up so that we could pretty easily make the actual backend a "pluggable" thing and you could potentially choose with a parameter or something whether to use free chatGPT like today, or the public conversations API, or chatGPT plus, and preserve the same shell experience across them all

**Reply by skytwosea** [reply]
Very cool. Thanks for this explanation! Looking forward to seeing how it develops.
